net-analyzer/nettop/nettop-0.2.3-r2.ebuild

```diff
@@ -19,12 +17,10 @@ DEPEND="
 "
 RDEPEND="${DEPEND}"
 
-src_prepare() {
-	epatch \
-		"${FILESDIR}"/${P}-gcc411.patch \
-		"${FILESDIR}"/${P}-offbyone.patch
-	tc-export CC
```

Comment:
you sure this isnt needed anymore? did you check with `USE="-native-symlinks?"`?